colearn/ml_interface.py

```diff
@@ -16,8 +16,36 @@
 #
 # ------------------------------------------------------------------------------
 import abc
+from enum import Enum
 from typing import Any, Optional
+
+import onnx
+import onnxmltools
+import sklearn
+import tensorflow as tf
+import torch
 from pydantic import BaseModel
+from tensorflow import keras
+
+model_classes_keras = (tf.keras.Model, keras.Model, tf.estimator.Estimator)
+model_classes_scipy = (torch.nn.Module)
+model_classes_sklearn = (sklearn.base.ClassifierMixin)
+
+
+def convert_model_to_onnx(model: Any):
+    """
+    Helper function to convert a ML model to onnx format
+    """
+    if isinstance(model, model_classes_keras):
+        return onnxmltools.convert_keras(model)
+    if isinstance(model, model_classes_sklearn):
+        return onnxmltools.convert_sklearn(model)
+    if 'xgboost' in model.__repr__():
```

Comment:
My machine does not have xgboost installed as it seems like some amount of work to do so, so this avoids having to have it installed to run the tests.